README.md

```diff
@@ -9,15 +9,17 @@ The `winston-enricher` log format adds additional app, distributed trace and spa
 
 For the latest information, please see [the New Relic docs](https://docs.newrelic.com/docs/logs/new-relic-logs/enable-logs-context/enable-logs-context-apm-agents).
 
-## Getting Started
-
-### Installation
+## Installation
 
 ```bash
 npm install @newrelic/winston-enricher
 ```
 
-### Usage
+## Getting Started
+
+>[Simple steps to start working with the software similar to a "Hello World"]
```

Comment:
It looks like this is incomplete?

Reply:
Yes, this is meant to be a prompt to get you all to add something there. If you'd rather not, simply delete that section. This is, I suppose, technically a required section, but I realize it may not apply for all of our repos.

Reply:
Ah okay, thanks, what's the recommended method for adding something?

Reply:
I don't know :) I'd recommend reaching out to Jodee Varney next week. She's the one spearheading this project.

Are there simple steps for getting started? If not, I think we can delete that section.

Reply:
Oops, looks like I forgot about this one.

Can we just delete the "Getting Started" section for now? Thanks!

Reply:
Sure! Just want to make sure that you're aware that this is a "required" section in the repo consistency project.  :)

Reply:
🤷 you could swap the "usage" with the "getting started" section but we have the content we have for now.

this repo was pre-recent standards. at some point more content can maybe be added but it is missing from this PR so just looking for a way to get this landed.

on the plus side, i can't imagine any developers are going to be confused either way.

Reply:
I think it's fine the way it is. No action necessary at the moment or possible ever :)

Reply:
Sounds good. Thanks :)